.yarnrc

```diff
@@ -0,0 +1 @@
+save-prefix ""
```

Comment:
what does this do?

Reply:
this is for when you do `yarn add some-dep`, it will add that dep as an exact version, so `"some-dep": "1.2.3"` instead of `"some-dep": "^1.2.3"`.

Reply:
oh nice